recipes/m4rie/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "20150908" %}
+{% set name = "m4rie" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/malb/m4rie/downloads/{{ name }}-{{ version }}.tar.gz
+  sha256: a0c3f46c399ed2e4af5aaad9de9db0962d15d5ccb9bd0e313df6b2c16bf8e0b1
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - libpng >=1.6.28,<1.7
+    - zlib 1.2.*
+    - m4ri
+  run:
+    - libpng >=1.6.28,<1.7
+    - zlib 1.2.*
+    - m4ri
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/m4rie/m4rie.h
+    - test -f ${PREFIX}/lib/lib{{ name }}.a
+    - test -f ${PREFIX}/lib/lib{{ name }}.dylib  # [osx]
+    - test -f ${PREFIX}/lib/lib{{ name }}.so     # [linux]
+
+about:
+  home: https://bitbucket.org/malb/m4rie
+  license: GPLv2+
+  license_file: {{ SRC_DIR }}/COPYING
```

Comment:
I thought that `{{ SRC_DIR }}` was not supposed to work with latest `conda`. That we needed `source_files` instead. But things are green so let's merge this.